packages/core/saved-objects/docs/openapi/paths/api@encrypted_saved_objects@_rotate_key.yaml

```diff
@@ -0,0 +1,52 @@
+post:
+  summary: Rotate the encryption key for encrypted saved objects.
+  operationId: rotateEncryptionKey
+  description: >
+    Superuser role required. This functionality is in technical preview and may be changed or removed in a future release. Elastic will apply best effort to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+    NOTE: In most cases, total will be greater than successful even if failed is zero. The reason is that Kibana may not need or may not be able to rotate encryption keys for all encrypted saved objects. 
+  tags:
+    - saved objects 
+  parameters:
+    - $ref: '../components/headers/kbn_xsrf.yaml'
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            batch_size:
+              description: Specifies a maximum number of saved objects that Kibana can process in a single batch. Bulk key rotation is an iterative process since Kibana may not be able to fetch and process all required saved objects in one go and splits processing into consequent batches. By default, the batch size is 10000, which is also a maximum allowed value.
+              type: number
+              default: 10000
+            type:
+              description: Limits encryption key rotation only to the saved objects with the specified type. By default, Kibana tries to rotate the encryption key for all saved object types that may contain encrypted attributes.
+              type: string
+           
+        examples:
+          rotateEncryptionKeyRequest:
+            $ref: '../components/examples/key_rotation_request.yaml'
```

Comment:
>            $ref: '../components/examples/key_rotation_request.yaml'

I think this is actually an example of the response body not the request body.  In which case, it should be renamed to something like "key_rotation_response.yaml" and referenced from the "responses" section instead of here.

Reply:
Per the results of my lint checks, the same is true of the example for the "resolve import errors" example too. If you want my help fixing, just lmk!  You can place these "response" examples in the same location as we did for the export_objects_response.yaml

Reply:
Good catches on both! The Rotate Key Encryption API doesn't even have a request body — so no example needed there.

I fixed this up in [`ce0a927` (#162566)](https://github.com/elastic/kibana/pull/162566/commits/ce0a92711355a47c47a505f1a9b4b97791ee7999)